openmp/runtime/src/CMakeLists.txt

```diff
@@ -108,7 +108,11 @@ else()
     # Unix specific files
     libomp_append(LIBOMP_CXXFILES z_Linux_util.cpp)
     libomp_append(LIBOMP_CXXFILES kmp_gsupport.cpp)
-    libomp_append(LIBOMP_GNUASMFILES z_Linux_asm.S) # Unix assembly file
+    if(${CMAKE_SYSTEM_NAME} MATCHES "AIX")
+      libomp_append(LIBOMP_GNUASMFILES z_AIX_asm.S) # AIX assembly file
```

Comment:
It looks like `z_AIX_asm.S` is missing in this patch

Reply:
@shiltian It says in the description that that portion would be added in a separate PR.

Reply:
Oh, sorry for missing that part. If that is the case, that PR needs to be landed before this one, otherwise it will cause issues. However, I don't know how that patch would be tested w/o this one. That being said, I think it might make more sense to include it in this patch as well.

Reply:
We plan to upstream changes in stages. This PR contains initial changes and lays the ground for subsequent AIX port PRs. The PR for `z_AIX_asm.S` will be posted shortly. Since the buildbot of AIX libomp is not set up yet and the code is guarded for AIX, this shouldn't cause problems. Our downstream build has LIT tests clean and we are still enabling affinity and some other things.